contracts/examples/libraries/Migrate_1_3_0_to_1_2_0.sol

```diff
@@ -2,28 +2,32 @@
 pragma solidity >=0.7.0 <0.9.0;
 import "../../libraries/SafeStorage.sol";
 
-/// @title Migration - migrates a Safe contract from 1.3.0 to 1.2.0
-/// @author Richard Meissner - <richard@gnosis.io>
+/**
+ * @title Migration - migrates a Safe contract from 1.3.0 to 1.2.0
+ * @author Richard Meissner - @rmeissner
+ */
 contract Migration is SafeStorage {
     bytes32 private constant DOMAIN_SEPARATOR_TYPEHASH = 0x035aff83d86937d35b32e04f0ddc6ff469290eef2f1b692d8a815c89404d4749;
 
     address public immutable migrationSingleton;
     address public immutable safe120Singleton;
 
     constructor(address targetSingleton) {
+        // Singleton address cannot be null.
```

Comment:
Nit pick: I would say `cannot be the Zero address` (even so I probably wrote the initial comment :see_no_evil: